scripts/install.dependencies.R

```diff
@@ -22,6 +22,8 @@ if(!("Rgraphviz" %in% installed.packages()[,"Package"])) {
 
 # install packages needed from CRAN
 list.of.packages <- c('abind', 'car', 'chron', 'coda', 'data.table', 'doSNOW', 'dplR', 'earth', 'emulator',
+                      'ggmap', 'ggplot2', 'gridExtra', 'Hmisc', 'kernlab',
```

Comment:
@tonygardella you appear to have duplicated a lot of packages rather than just adding a package or two to those lines -- please clean this up so packages aren't listed multiple times.